modules/op-defining-and-creating-pipelineresources.adoc

```diff
@@ -0,0 +1,77 @@
+// Ths module is included in the following assembly:
+//
+// *openshift_pipelines/op-creating-applications-with-cicd-pipelines.adoc
+
+[id="defining-and-creating-pipelineresources_{context}"]
+= Defining and creating Pipeline Resources
+
+_Pipeline Resources_ are artifacts that are used as inputs to a Task and can be an output of a Task.
```

Comment:
nit: Shouldn't it be `… are used as inputs or outputs of a Task` for less repetition ?

Reply:
done